docs/contributing/commit-discipline.md

```diff
@@ -279,6 +279,23 @@ might worry about it breaking.
   you developed this commit or pull request, like "First I tried X" or
   "I changed Y".
 
+#### Mentioning other contributors
+
+You can
+[credit](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors)
+co-authors on a commit by adding a `Co-authored-by:` line after a blank line at
+the end of your commit message:
+
+    Co-authored-by: Greg Price <greg@zulip.com>
+
+You can also add other notes, such as `Reported-by:`, `Debugged-by:` or
+`Suggested-by:`, but we do not generally do so in this project.
```

Comment:
The end of this reads to me like a bit of an admonition not to do it, which I don't think we intend. Perhaps:
```suggestion
You can also add other notes, such as `Reported-by:`, `Debugged-by:`, or
`Suggested-by:`, but we don't typically do so.
```